src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -159,6 +159,19 @@ embed.uno-frameworkelement.uno-unarranged {
     cursor: text;
 }
 
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover,
+textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+  /* Workaround based on https://stackoverflow.com/a/68240841 */
+  transition: background-color 600000s 0s, color 600000s 0s, background-image 600000s 0s, appearance 600000s 0s;
+}
```

Comment:
@carldebilly

Reply:
That's ugly hack...  I can't see how it could break anything, so I'm ok with it as soon the problem is properly referenced as it is now.